File: PHP/CreditCard.php
<?php
require_once('Payment.php');
    class CreditCard extends Payment {
        public $cardNumber;
        public $cvv;
        public $expirationDate;
        public $ammount;

        public function __construct($id, $cardNumber, $cvv, $expirationDate, $ammount){
            parent::__construct($id);
            $this ->cardNumber = $cardNumber;
            $this ->cvv = $cvv;
            $this ->expirationDate = $expirationDate;
            $this ->ammount = $ammount;
        } 

    }

?>

File: Python/CreditCard.py
from payment import Payment

class CreditCard(Payment):
    cardNumber     = int
    cvv            = int
    dateExpiration = str
    ammount        = str

    def __init__(self, id, cardNumber, cvv, dateExpiration, ammount):
        super().__init__(id)
        self.cardNumber = cardNumber
        self.cvv = cvv
        self.dateExpiration = dateExpiration
        self.ammount = ammount